CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
+1. Fork the repository.
+1. Create an `upstream` remote.
+```bash
+$ git remote add upstream git@github.com:fastly/terraform-provider-fastly.git
+```
+1. Create a feature branch.
+1. Make changes.
+1. Write tests.
+1. Validate your change via the steps documented [in the README](./README.md#testing).
+1. Open a pull request against `upstream master`.
```

Comment:
Suggest dropping the remote name here to avoid confusion with the branch name.

Reply:
Yeah, we could probably assume that people would know that they would need to open a PR against the Fastly repo.

Otherwise, if they open a PR against just `master`, it will default to `origin master`, which would be their fork.

Reply:
Yep, as discussed before, I'm an idiot.  Carry on.